### PR TITLE
feat #148 : Apprenticeship page

### PR DESCRIPTION
**Pull request by keraliss**
Screenshot from 2024-06-10 22-10-06
Screenshot from 2024-06-10 22-09-59
Screenshot from 2024-06-10 22-06-39
Screenshot from 2024-06-10 22-09-56
Screenshot from 2024-06-10 22-09-51
Screenshot from 2024-06-10 22-06-45
Screenshot from 2024-06-10 22-09-48
Screenshot from 2024-06-10 22-06-51
Screenshot from 2024-06-10 22-09-44
Screenshot from 2024-06-10 22-06-59
Screenshot from 2024-06-10 22-09-40
Screenshot from 2024-06-10 22-09-36
Screenshot from 2024-06-10 22-07-02
Screenshot from 2024-06-10 22-07-05
Screenshot from 2024-06-10 22-07-05
Screenshot from 2024-06-10 22-07-02
Screenshot from 2024-06-10 22-09-36
Screenshot from 2024-06-10 22-06-59
Screenshot from 2024-06-10 22-09-40
Screenshot from 2024-06-10 22-06-51
Screenshot from 2024-06-10 22-09-44
Screenshot from 2024-06-10 22-09-48
Screenshot from 2024-06-10 22-06-45
Screenshot from 2024-06-10 22-09-51
Screenshot from 2024-06-10 22-09-56
Screenshot from 2024-06-10 22-06-39
Screenshot from 2024-06-10 22-09-59
Screenshot from 2024-06-10 22-06-35
Screenshot from 2024-06-10 22-10-06

### CODE REVIEW

**Comment by keraliss** [comment]
Sorry the order of images got messed up. if you find it difficult, ping me, and i'll re-upload the images

File: public/apprenticeship/bitlogo.png
Comment (emjshrx) [inline public/apprenticeship/bitlogo.png:1]: Dont we already have logos, can we reuse them?

File: public/apprenticeship/readingClub.png
Comment (emjshrx) [inline public/apprenticeship/readingClub.png:1]: Too big image, webp?

File: public/apprenticeship/reviewClub.png
Comment (emjshrx) [inline public/apprenticeship/reviewClub.png:1]: Too big image, webp?

File: public/apprenticeship/bitlogo.png
Comment (keraliss) [reply public/apprenticeship/bitlogo.png:1]: the logos we had were a bit different than what is in the design. i wanted these to be as close to the design as possible, so i opted to use the exact ones in the design. these are very small files, so don't think will hamper the performance!

**Comment by rajarshimaitra** [comment]
ACK from the pics. We need to also open up the application form with it, and connect it to a google sheet. @keraliss

**Review by emjshrx — APPROVED** [review]

**Pull request merged**